src/components/Match/Laning/index.jsx

```diff
@@ -20,17 +20,17 @@ class Laning extends React.Component {
     this.setState({ ...this.state, selectedPlayer: playerSlot });
   }
   render() {
-    const { match } = this.props;
+    const players = [...this.props.match.players];
```

Comment:
this won't be sufficient by itself since this is the not the only table that sorts.  Probably this needs to occur within the Table component's sorting logic.